qa/tasks/radosgw_admin.py

```diff
@@ -1106,6 +1106,40 @@ def task(ctx, config):
     # TESTCASE 'zonegroup-info', 'zonegroup', 'get', 'get zonegroup info', 'succeeds'
     (err, out) = rgwadmin(ctx, client, ['zonegroup', 'get'], check_status=True)
 
+    # TESTCASE 'add-storage-class', 'zonegroup', 'add', 'add storage class', 'succeeds'
+    storage_class = 'GLACIER'
+    (err, out) = rgwadmin(ctx, client, ['zonegroup', 'placement', 'add', '--rgw-zonegroup', 'default', '--placement-id', 'default-placement', '--storage-class', storage_class], check_status=True)
+    assert err
```

Comment:
why this assert? its causing test failures like http://qa-proxy.ceph.com/teuthology/cbodley-2024-02-01_00:17:29-rgw-wip-cbodley2-testing-distro-default-smithi/7541364/teuthology.log:
```
2024-02-01T20:45:22.587 INFO:teuthology.orchestra.run.smithi050.stdout:[{"key":"default-placement","val":{"name":"default-placement","tags":[],"storage_classes":["GLACIER","STANDARD"]}}]
2024-02-01T20:45:22.587 DEBUG:tasks.util.rgw: json result: [{'key': 'default-placement', 'val': {'name': 'default-placement', 'tags': [], 'storage_classes': ['GLACIER', 'STANDARD']}}]
2024-02-01T20:45:22.588 ERROR:teuthology.run_tasks:Saw exception from tasks.
Traceback (most recent call last):
  File "/home/teuthworker/src/git.ceph.com_teuthology_d9fdb2209e15b39d9f061fd85399f352ce0f0894/teuthology/run_tasks.py", line 105, in run_tasks
    manager = run_one_task(taskname, ctx=ctx, config=config)
  File "/home/teuthworker/src/git.ceph.com_teuthology_d9fdb2209e15b39d9f061fd85399f352ce0f0894/teuthology/run_tasks.py", line 83, in run_one_task
    return task(**kwargs)
  File "/home/teuthworker/src/git.ceph.com_ceph-c_c5906db12c43b47f794ca8c271ee311532526b64/qa/tasks/radosgw_admin.py", line 1113, in task
    assert err
AssertionError
```